<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Research - Xiangtao Li</title>
  <meta name="description" content="Academic webpage of Prof. Xiangtao Li">
  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="/research/">
  <link rel="shortcut icon" type ="image/x-icon" href="/favicon.ico">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">

  <link rel="preconnect" href="https://player.vimeo.com">
  <link rel="preconnect" href="https://i.vimeocdn.com">
  <link rel="preconnect" href="https://f.vimeocdn.com">



<!-- Google Analytics (original) -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');

</script>

<!-- Global site tag (gtag.js) - Google Analytics 4 -->
<script async src="https://www.googletagmanager.com/gtag/js?id="></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', '');
</script>

<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','');</script>
<!-- End Google Tag Manager -->



</head>


  <body>

    <!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id="
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->

<nav class="navbar sticky-top navbar-expand-md navbar-dark bg-dark">
    <a class="navbar-brand" href="/">
     <img src="/favicon.ico" width="30" height="30" style="margin-right:5px" class="d-inline-block align-top" alt="">
      Xiangtao Li
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarColor02">
        <ul class="navbar-nav mr-auto">
        <ul class="navbar-nav">
          <li class="nav-item">
              <a class="nav-link" href="/">Home</a>
          </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/team">Team</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/publications">Publications</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/research">Research</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/adm. req.">Adm. req.</a>
           </li> 
          
        </ul>
  </div>
</nav>



    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12 col-xs-12">
  <style>
img{
  border-radius: 10px;
}
.col-md-3 {
  margin-top:10px;
  margin-bottom:10px;
  padding:0px;
  display:block;
  overflow:hidden;
  text-align:center;
  display: table-cell;
  background: white;
  border-radius: 20px;
  height: auto;
}
iframe {
  margin:0;
  padding:0;
  width: 175px;
  display: inline;
  vertical-align: middle;
}
.img-responsive {
    max-width: 100%;
    height: auto;
    margin-left: 10px;
    margin-right: 20px;
  }
</style>

<h2 id="research">Research</h2>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4> Distribution-Agnostic Deep Learning Enables Accurate Single-Cell Data Recovery and Transcriptional Regulation Interpretation</h4>

    <div class="rowl1">
      <p><img src="/images/research/su.png" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>We developed Bis, a distribution-agnostic deep learning model for accurately recovering missing sing-cell gene expression from multiple platforms. Bis is an optimal transport-based autoencoder model that can capture the intricate distribution of scRNA-seq data while addressing the characteristic sparsity by regularizing the cellular embedding space. Additionally, they propose a module using bulk RNA-seq data to guide reconstruction and ensure expression consistency. Experimental results show Bis outperforms other models across simulated and real datasets, showcasing superiority in various downstream analyses including batch effect removal, clustering, differential expression analysis, and trajectory inference. Moreover, Bis successfully restores gene expression levels in rare cell subsets in a tumor-matched peripheral blood dataset, revealing developmental characteristics of cytokine-induced natural killer cells within a head and neck squamous cell carcinoma microenvironment.</p>

      <p><em>Y. Su,  Z. Yu, Y. Yang, Y. Huang, K. Wong, X. Li*</em></p>
      <h5 id="nature-communications2023">Nature Communications(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4> Dynamic characterization and interpretation for proteinâ€“RNA interactions across diverse cellular conditions using HDRNet</h4>

    <div class="rowl1">
      <p><img src="/images/research/HDRNet.png" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>Here, we develop HDRNet, an end-to-end deep learning-based framework to precisely predict dynamic RBP binding events under diverse cellular conditions. Our results demonstrate that HDRNet can accurately and efficiently identify binding sites, particularly for dynamic prediction, outperforming other state-of-the-art models on 261 linear RNA datasets from both eCLIP and CLIP-seq, supplemented with additional tissue data. Moreover, we conduct motif and interpretation analyses to provide fresh insights into the pathological mechanisms underlying RNA-RBP interactions from various perspectives. Our functional genomic analysis further explores the gene-human disease associations, uncovering previously uncharacterized observations for a broad range of genetic disorders.</p>

      <p><em>H. Zhu, Y. Yang, Y. Wang, F. Wang, Y. Huang, Y. Chang, K. Wong*, X. Li*</em></p>
      <h5 id="nature-communications2023-1">Nature Communications(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4> Topological Identification and Interpretation for Single-cell Gene Regulation Elucidation across Multiple Platforms using scMGCA</h4>

    <div class="rowl1">
      <p><img src="/images/research/yu.webp" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>We develop a deep graph learning method, scMGCA, for single-cell data analysis. scMGCA is based on a graph-embedding autoencoder that simultaneously learns cell-cell topology representation and cluster assignments. We show that scMGCA is accurate and effective for cell segregation and batch effect correction, outperforming other state-of-the-art models across multiple platforms. In addition, we perform genomic interpretation on the key compressed transcriptomic space of the graph-embedding autoencoder to demonstrate the underlying gene regulation mechanism. We demonstrate that in a pancreatic ductal adenocarcinoma dataset, scMGCA successfully provides annotations on the specific cell types and reveals differential gene expression levels across multiple tumor-associated and cell signalling pathways.</p>

      <p><em>Z. Yu, Y. Su, Y. Lu, F. Wang, S. Zhang, Y. Chang, K. Wong*, X. Li*</em></p>
      <h5 id="nature-communications2023-2">Nature Communications(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Reliable Identification and Interpretation of Single-cell Molecular Heterogeneity and Transcriptional Regulation using Dynamic Ensemble Pruning</h4>

    <div class="rowl1">
      <p><img src="/images/research/DEPF.png" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>A dynamic ensemble pruning framework (DEPF) is proposed to identify and interpret single-cell molecular heterogeneity.  In particular, a silhouette coefficient-based indicator is developed and evaluated to determine the optimization direction of the bi-objective function. Following that, a bi-objective fruit fly optimization algorithm is designed to prune dynamically the low-quality basic clustering in the ensemble.</p>

      <p><em>Y. Fan, Y. Wang, F. Wang, L. Huang, Y. Yang, K. Wong, X. Li*</em></p>
      <h5 id="advanced-science2023">Advanced Science(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Enabling Single-cell Drug Response Annotations from Bulk RNA-seq using SCAD</h4>

    <div class="rowl1">
      <p><img src="/images/research/SCAD.jpg" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>A transfer learning model is proposed to infer the drug sensitivities at single-cell level. This framework learns bulk transcriptome profiles and pharmacogenomics information from population cell lines in a large public dataset and transfers the knowledge to infer drug efficacy of individual cells. The results suggest that it is suitable to learn knowledge from pre-clinical cell lines to infer pre-existing cell subpopulations with different drug sensitivities prior to drug exposure. In addition, the model offers a new perspective on drug combinations. It is observed that drug-resistant subpopulation can be sensitive to other drugs (e.g., a subset of JHU006 is Vorinostat-resistant while Gefitinib-sensitive); such finding corroborates the previously reported drug combination (Gefitinib + Vorinostat) strategy in several cancer types. The identified drug sensitivity biomarkers reveal insights into the tumor heterogeneity and treatment at cellular resolution.</p>

      <p><em>Z. Zheng, J. Chen, X. Chen, L. Huang, W. Xie, Q. Lin, X. Li*, K. Wong*</em></p>
      <h5 id="advanced-science2023-1">Advanced Science(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>A lightweight framework for chromatin loop detection on single-cell Hi-C</h4>

    <div class="rowl1">
      <p><img src="/images/research/Hi-C.jpg" class="img-responsive" style="float: left; border-radius: 5px; width: 280px; height: 238px" /></p>

      <p>A lightweight framework called scGSLoop is proposed, which sets a new paradigm for scHi-C loop calling by adapting the training and inferencing strategies of graph-based deep learning to leverage the sequence features and 1D positional information of genomic loci. With this framework, sparsity is no longer a challenge, but rather an advantage that the model leverages to achieve unprecedented computational efficiency. Compared to existing methods, scGSLoop makes more accurate predictions and is able to identify more loops that have the potential to play regulatory roles in genome functioning. Moreover, scGSLoop preserves single-cell information by identifying a distinct group of loops for each individual cell, which not only enables an understanding of the variability of chromatin looping states between cells, but also allows scGSLoop to be extended for the investigation of multi-connected hubs and their underlying mechanisms.</p>

      <p><em>F. Wang, H. Alinejad-Rokny, J. Lin, T. Gao, X. Chen, L. Meng, X. Li*, K. Wong*</em></p>
      <h5 id="advanced-science2023-2">Advanced Science(2023)</h5>

      <ul style="overflow: hidden">
  </ul>

    </div>

  </div>
</div>

</div>

      </div>
    </div>

    <br/>
<section id="footer">
<div class="container-footer">
  <div class="panel-footer">
	  <div class="row">
		<div class="col-sm-4">
		    <h5>About</h5>	
            <p>Xiangtao Li<br/> Professor of CS<br/> Jilin University
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Contact</h5>	
            <p><a href="mailto:lixt314@jlu.edu.cn" target="_blank"><i class="fa fa-envelope fa-1x"></i> Contact Xiangtao via email</a> <br/>
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Coordinates</h5>	
            <p>Jilin University<br/> Changchun, Jilin
</p>
		</div>
	  </div>

      <center><p>&copy 2024 Xiangtao Li </p></center>
	</div>
  </div>
</div>

<script src="/assets/javascript/bootstrap/jquery.min.js"></script>
<script src="/assets/javascript/bootstrap/bootstrap.bundle.min.js"></script>


  </body>

</html>
